### PR TITLE
adds redirect to workflow frontend

### PR DESCRIPTION
**Pull request by normanrz**
Adds a redirect from `/workflows` to `https://workflows.voxelytics.com/workflows` (with sub paths). With this we can already use webKnossos-hosted URLs to point to workflow reports, before it is integrated into webKnossos itself.

------
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/router.tsx [hunk 1]
```diff
@@ -100,6 +100,16 @@ function PageNotFoundView() {
   );
 }
 
+function RedirectToWorkflowViewer() {
+  const location = useLocation();
+
+  useEffect(() => {
+    windowLocation.assign(`https://workflows.voxelytics.com${location.pathname}${location.search}`);
```
Comment (philippotto) [inline frontend/javascripts/router.tsx:107]: The final URL will be `https://workflows.voxelytics.com/workflows/...` then, right? It's a bit redundant, but probably acceptable. Alternatively, you could trim the leading `workflows` from the pathname.

**Review by philippotto — APPROVED** [review]
lgtm :+1:

File: frontend/javascripts/router.tsx (hunk 1 above)
Comment (normanrz) [reply frontend/javascripts/router.tsx:107]: Well the final URL will be `https://webknososs.org/workflows` ;-) I'd like to keep the URL schemes as close as possible, which introduces this (hopefully temporary) redundancy.

Comment (philippotto) [reply frontend/javascripts/router.tsx:107]: ok, makes sense :+1:

**Pull request merged**